qadence/draw/utils.py

```diff
@@ -143,13 +143,11 @@ def _(circuit: QuantumCircuit, *args: Any, **kwargs: Any) -> QuantumCircuitDiagr
 
 @make_diagram.register
 def _(model: QuantumModel, *args: Any, **kwargs: Any) -> QuantumCircuitDiagram:
-    raise ValueError("Not yet supported.")
-
     # FIXME: include measurement icon
-    if model.out_features > 1:
+    if model.out_features > 1:  # type: ignore
```

Comment:
Did you get `mypy` warnings ?

Reply:
yes, good point actually, I'll catch the `model.out_features = None` case.